### PR TITLE
Fixes #19768 - Fix Layout/ClosingParenthesisIndentation cop

### PR DESCRIPTION
**Pull request by mmoll**

### CODE REVIEW

**Comment by mmoll** [comment]
While this one is maybe not so nice as such, it's IMHO a needed groundwork for further mutliline cops, which can get enabled later.

vote :+1: for merging and enabling this cop
vote :-1: for disabling this cop

**Comment by mmoll** [comment]
rebased, more votes, please :)

**Comment by mmoll** [comment]
Do we want to get this in or not? :point_up:

File: test/controllers/api/v2/hosts_controller_test.rb [hunk 1]
```diff
@@ -967,7 +967,7 @@ def setup
       cr = FactoryBot.create(:compute_resource, :vmware, :uuid => 'Solutions',
```
Comment (ekohl) [inline test/controllers/api/v2/hosts_controller_test.rb]: If you're changing the closing indent, you should also fix this line. Looks like there's a space too much

Comment (mmoll) [reply test/controllers/api/v2/hosts_controller_test.rb]: I did change this manually now, but that would be the job of a (yet to be added) multi line cop.

**Comment by mmoll** [comment]
rebased.

**Comment by mmoll** [comment]
rebased. anybody?

**Comment by mmoll** [comment]
rebased.

**Review by tbrisker — APPROVED** [review]
Thanks @mmoll !

**Pull request merged**